pkg/controller/astarte/reconcile/utils.go

```diff
@@ -400,6 +400,10 @@ func getDeploymentStrategyForClusteredResource(cr *apiv1alpha1.Astarte, resource
 		return appsv1.DeploymentStrategy{
 			Type: appsv1.RecreateDeploymentStrategyType,
 		}
+	case component == apiv1alpha1.TriggerEngine:
```

Comment:
join it together with the previous case.

`case component == apiv1alpha1.DataUpdaterPlant, component == apiv1alpha1.TriggerEngine:`